### PR TITLE
Key exchange

### PR DESCRIPTION
**Pull request by podhrmic**
Messages needed for key exchange management.
Added SECURE_LINK_STATUS to monitor its status.

### CODE REVIEW

**Comment by podhrmic** [comment]
this is needed to get https://github.com/paparazzi/paparazzi/pull/2205 to work

File: message_definitions/v1.0/messages.xml [hunk 1]
```diff
@@ -2478,6 +2486,12 @@
       <field name="desired_sigma" type="int16">Desired angle to have w.r.t. the neighbor</field>
     </message>
 
+    <message name="KEY_EXCHANGE_GCS" id="159">
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml]: what is the reason for not having the `link` attribute (probably set to `forwarded`) nor the `ac_id` field ?

Comment (podhrmic) [reply message_definitions/v1.0/messages.xml]: Good point. I must have missed that (I am using it with the Rust link so it worked as is). Will fix.

Comment (podhrmic) [reply message_definitions/v1.0/messages.xml]: Fixed

**Pull request merged**